pkg/actions/nodegroup/get_test.go

```diff
@@ -1,10 +1,12 @@
 package nodegroup_test
```

Comment:
Can we have at least some of the error scenarios tested too please? :) Pretty please? :D